packages/docusaurus-theme-translations/locales/pl/theme-search-algolia.json

```
@@ -1,29 +1,29 @@
 {
   "theme.SearchBar.label": "Szukaj",
   "theme.SearchBar.seeAll": "Wyświetl wszystkie {count} wyników",
-  "theme.SearchModal.errorScreen.helpText": "You might want to check your network connection.",
-  "theme.SearchModal.errorScreen.titleText": "Unable to fetch results",
-  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape key",
-  "theme.SearchModal.footer.closeText": "to close",
-  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Arrow down",
-  "theme.SearchModal.footer.navigateText": "to navigate",
-  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Arrow up",
-  "theme.SearchModal.footer.searchByText": "Search by",
-  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter key",
-  "theme.SearchModal.footer.selectText": "to select",
-  "theme.SearchModal.noResultsScreen.noResultsText": "No results for",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Let us know.",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Believe this query should return results?",
-  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Try searching for",
-  "theme.SearchModal.placeholder": "Search docs",
-  "theme.SearchModal.searchBox.cancelButtonText": "Cancel",
-  "theme.SearchModal.searchBox.resetButtonTitle": "Clear the query",
-  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Favorite",
-  "theme.SearchModal.startScreen.noRecentSearchesText": "No recent searches",
-  "theme.SearchModal.startScreen.recentSearchesTitle": "Recent",
-  "theme.SearchModal.startScreen.removeFavoriteSearchButtonTitle": "Remove this search from favorites",
-  "theme.SearchModal.startScreen.removeRecentSearchButtonTitle": "Remove this search from history",
-  "theme.SearchModal.startScreen.saveRecentSearchButtonTitle": "Save this search",
+  "theme.SearchModal.errorScreen.helpText": "Może warto sprawdzić połączenie sieciowe.",
```

```suggestion
  "theme.SearchModal.errorScreen.helpText": "Sprawdź swoje połączenie sieciowe.",
```

The base sentence is a recommendation, let's stick to that tone.